File: cases/taint/c/infeasible-branch-negative/infeasible_branch_negative.c
int dfb_source(void) { // DFB-SOURCE: infeasible-branch-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: infeasible-branch-sink

void run(void) {
    int value = 0;
    if (0) {
        value = dfb_source(); // DFB-WITNESS: infeasible-tainted-branch
    }
    dfb_sink(value);
}

File: cases/taint/rust/infeasible-branch-negative/infeasible_branch_negative.rs
fn dfb_source() -> i32 { // DFB-SOURCE: infeasible-branch-input
    1
}

fn dfb_sink(value: i32) {} // DFB-SINK: infeasible-branch-sink

fn run() {
    let mut value = 0;
    if false {
        value = dfb_source(); // DFB-WITNESS: infeasible-tainted-branch
    }
    dfb_sink(value);
}
